dev/devicelab/bin/tasks/hot_mode_dev_cycle__benchmark.dart

```diff
@@ -10,18 +10,22 @@ import 'package:flutter_devicelab/framework/adb.dart';
 import 'package:flutter_devicelab/framework/framework.dart';
 import 'package:flutter_devicelab/framework/utils.dart';
 
-void main() {
+void main({bool isPreviewDart2: false}) {
```

Comment:
here and elsewhere: please put spaces around named argument declarations (inside the braces).

Reply:
Done